corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -950,6 +951,7 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                 startPage: paginateItems.startPage,
                 title: title.trim(),
                 description: description === undefined ? "" : markdown.render(description.trim()),
+                selectText: this.selectText === undefined ? "" : this.selectText,
```

Comment:
we should wrap `this.selectText` in `markdown.render` to sanitize it

Reply:
I think this is probably a good idea, but I wonder if it would be redundant with the underscore html escaping [here](https://github.com/dimagi/commcare-hq/pull/33976/commits/38bd3281708f97b3f1a8ee4729d2f4a4918896a8)? @orangejenny what do you think?